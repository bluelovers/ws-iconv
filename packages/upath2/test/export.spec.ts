import _m0 = require('../');
import _m1 from '../';

test(`import check`, () =>
{

	let actual = Object.keys(_m0);
	let expected = Object.keys(_m1);

	expect(actual).toStrictEqual(expected);
	//expect(actual).toBeInstanceOf(Date);
	expect(actual).toMatchSnapshot();

});
