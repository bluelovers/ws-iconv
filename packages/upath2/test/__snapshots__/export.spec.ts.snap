// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import check 1`] = `
Array [
  "sep",
  "name",
  "fn",
  "PathWrap",
  "win32",
  "posix",
  "upath",
  "default",
]
`;
